README.md

```diff
@@ -23,6 +23,8 @@ Now you can run the script by typing (You will need netcat installed on the comp
 
     ./init_and_wait_for_db.sh
 
+**You will need to modify config.example.py with the correct database parameters.**
```

Comment:
Oh crud, you are right.

Changed the env name from the default to `FAF_DB_PASSWORD`. Proper fix is to make the db container expose this.